crates/cxx-qt-gen/src/writer/cpp/mod.rs

```diff
@@ -507,6 +535,15 @@ mod tests {
           // setter
         }
 
+        void MyObject::privateMethod() const {
```

Comment:
move `{` to new line

Reply:
Already answered in the first comment :)